buildbot/osuosl/master/config/builders.py

```diff
@@ -2020,7 +2020,8 @@
                             "-DTEST_SUITE_SOLLVEVV_OFFLOADING_CFLAGS=-fopenmp;-fopenmp-targets=amdgcn-amd-amdhsa;-Xopenmp-target=amdgcn-amd-amdhsa;-march=gfx906",
                             "-DTEST_SUITE_SOLLVEVV_OFFLOADING_LDLAGS=-fopenmp;-fopenmp-targets=amdgcn-amd-amdhsa;-Xopenmp-target=amdgcn-amd-amdhsa;-march=gfx906",
                         ],
-                        add_lit_checks=["check-offload", "check-libc-amdgcn-amd-amdhsa"]
+                        add_lit_checks=["check-offload", "check-libc-amdgcn-amd-amdhsa"],
+                        add_openmp_lit_args=["--filter-out=offloading/pgo1.c"]
```

Comment:
```suggestion
                        add_openmp_lit_args=["--filter-out=offloading/pgo1.c"],
```
to avoid further spurious line changes when adding options